CHANGELOG.md

```diff
@@ -593,6 +593,7 @@
 - [Added `Table.format` for the in-memory backend.][8150]
 - [Implemented truncate `Date_Time` for database backend (Postgres only).][8235]
 - [Initial Enso Cloud APIs.][8006]
+- [Renamed `Table.replace` to `text_replace` and renamed `Table.lookup_and_replace` to `Table.merge`][8564]
```

Comment:
nitpick because we are not fully consistent within this file, but the Libs section usually tries to end the entries with a dot:
```suggestion
- [Renamed `Table.replace` to `text_replace` and renamed `Table.lookup_and_replace` to `Table.merge`.][8564]
```